site/content/en/docs/Guides/feature-stages.md

```diff
@@ -47,11 +46,10 @@ The current set of `alpha` and `beta` feature gates:
 |-----------------------------------------------------------------------------------------------------------------------|--------------------------------|----------|---------|--------|
 | [Custom resync period for FleetAutoscaler](https://github.com/googleforgames/agones/issues/1955)                      | `CustomFasSyncInterval`        | Enabled  | `Beta`  | 1.25.0 |
 | [GameServer Stable Network ID]({{% ref "/docs/Reference/gameserver.md#stable-network-id" %}})                         | `PodHostname`                  | Enabled  | `Beta`  | 1.32.0 |
-| [Reset Metric Export on Fleet / Autoscaler deletion]({{% relref "./metrics.md#dropping-metric-labels" %}})            | `ResetMetricsOnDelete`         | Enabled  | `Beta`  | 1.26.0 |
 | [Graceful Termination for GameServer SDK](https://github.com/googleforgames/agones/pull/2205)                         | `SDKGracefulTermination`       | Enabled  | `Beta`  | 1.18.0 |
-| [GameServer state filtering on GameServerAllocations](https://github.com/googleforgames/agones/issues/1239)           | `StateAllocationFilter`        | Enabled  | `Beta`  | 1.26.0 |
 | [GameServer player capacity filtering on GameServerAllocations](https://github.com/googleforgames/agones/issues/1239) | `PlayerAllocationFilter`       | Disabled | `Alpha` | 1.14.0 |
 | [Player Tracking]({{< ref "/docs/Guides/player-tracking.md" >}})                                                      | `PlayerTracking`               | Disabled | `Alpha` | 1.6.0  |
+| [Reset Metric Export on Fleet / Autoscaler deletion]({{% relref "./metrics.md#dropping-metric-labels" %}})            | `ResetMetricsOnDelete`         | Disabled | `Alpha` | 1.26.0 |
```

Comment:
I think you accepted the wrong side of a merge conflict here - `ResetMetricsOnDelete` should be in `Beta` as of 1.32 yeah?